AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -612,9 +613,10 @@ - (void)channel:(id<MSChannelProtocol>)__unused channel willSendLog:(id<MSLog>)l
     if ([logObject isKindOfClass:[MSAppleErrorLog class]]) {
       MSAppleErrorLog *appleErrorLog = static_cast<MSAppleErrorLog *>(log);
       MSErrorReport *report = [MSErrorLogFormatter errorReportFromLog:appleErrorLog];
-      dispatch_async(dispatch_get_main_queue(), ^{
-        [delegate crashes:self willSendErrorReport:report];
-      });
+      NSObject* delegateInstance = static_cast<NSObject *>(delegate);
+      [MSUtility performSelectorOnMainThread:delegateInstance
+                                withSelector:@selector(crashes:willSendErrorReport:)
+                                 withObjects:self, report, [NSNull null]];
```

Comment:
Why do we need `NSNull` here?

Reply:
Null allows to determine the end of the arguments passed